### PR TITLE
Upgrade to babel@7

### PR DESCRIPTION
**Pull request by Andarist**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,13 +47,7 @@
     "compile:commonjs": {
       "command": "babel -d lib/ src/ && ncp ./src/index.d.ts ./lib/index.d.ts",
       "env": {
-        "NODE_ENV": "commonjs"
-      }
-    },
-    "compile:umd": {
```
Comment (Andarist) [inline package.json]: personally I would move all builds (umd, umd:min, cjs, es) to rollup, but not sure how do u feel about it

**Comment by markerikson** [comment]
Seems like this still is relevant.  @Andarist , any chance you could update the PR to merge cleanly?

**Comment by Andarist** [comment]
@markerikson I've rebased and confirmed that it still compiles but please re-check the current setup.

**Comment by Andarist** [comment]
Oh, I haven't spotted in a rush that this repo is not using Travis any longer 😅

**Comment by markerikson** [comment]
that's because I only merged the other PR like 10 minutes ago :)

**Comment by markerikson** [comment]
Well, this seems to build cleanly.  Let's get it in and move things along.  Thanks, @andarist !

**Pull request merged**